portfolio/src/main/webapp/script.js

```diff
@@ -46,6 +46,21 @@ function getJSONString() {
   });
 }
 
+function loadingMyComments() {
+    // Stats is an object, not a string, so we have to
+    // reference its fields to create HTML content.
+    document.getElementById('log').innerHTML = "";
+    let elem = Integer.parseInt(document.getElementById("amount").value);
+    if (elem != 0) {
+      fetch('/data').then(response => response.json()).then((jsonVersion) => {
+        const oneComment = document.getElementById('log');
+        for (let i=0; i<elem; i++) {
```

Comment:
Nit: add spaces around operators

Reply:
Just resolved all nits and fixed some functionality issues that I had before, thank you for the comments let me know if anything else needs work!